eng/targets/Helix.targets

```diff
@@ -106,14 +106,8 @@ Usage: dotnet msbuild /t:Helix src/MyTestProject.csproj
       <HelixContent Include="$(OutputPath)/Microsoft.VisualStudio.TestPlatform.Extension.Xunit.Xml.TestLogger.dll" />
     </ItemGroup>
 
-    <!--
-      Could use _GetPackageVersionInfo target (defined in eng/targets/Packaging.targets and included in every C#
-      and F# project) instead of the $(SharedFxVersion) property but the property works everywhere except in site
-      extensions projects. Could also use $(TargetingPackVersion) but that has slightly more complicated semantics
-      and doesn't keep up when in servicing.
-    -->
     <ItemGroup Condition=" '$(TestDependsOnAspNetRef)' == 'true' AND '$(IsTargetingPackBuilding)' == 'true' ">
-      <HelixContent Include="$(RepoRoot)artifacts\packages\Release\Shipping\Microsoft.AspNetCore.App.Ref.$(SharedFxVersion).nupkg" />
+      <HelixContent Include="$(RepoRoot)artifacts\packages\Release\Shipping\Microsoft.AspNetCore.App.Ref.$(TargetingPackVersion).nupkg" />
```

Comment:
Nit: can we put this in a variable to use in both here and the nupkg file we pass below into the commands?   
`Microsoft.AspNetCore.App.Ref.$(TargetingPackVersion).nupkg`

Reply:
Looks like I need to rebase anyhow. So, sure, I'll make another update here.

But, I'll go w/ your earlier suggestion of removing the `*.nupkg` names from the command line and instead use the passed `$(SharedFxVersion)` to craft the names on the Helix agent.

Reply:
Nice, a few less arguments is much better since its already getting out of control with how many we pass!